### ISSUE TITLE
High CPU usage on windows 10

### ISSUE THREAD

**Issue by whitecolor** [meganz/MEGAsync#67]
MegaSync gets very high CPU usage, though it is not actually performing any sync work at the moment.
![image](https://cloud.githubusercontent.com/assets/736697/22511542/d20854e8-e8b7-11e6-8bd7-fa447ca269b0.png)

![image](https://cloud.githubusercontent.com/assets/736697/22511595/02dcf380-e8b8-11e6-8e70-0bd9965f33f2.png)

If I put on pause it doesn't change the CPU consumption.

**Comment by anacondaqq** [comment]
Update your client to latest version.
![](https://i.imgur.com/zB5wFMQ.png)
![](https://i.imgur.com/HhTJIfs.png)

Can't confirm.

**Comment by mateusz-bajorek** [comment]
I can confirm that the issue is still present. When I close the MEGAsync and reopen it the CPU usage is low but then aftrer some time there is what follows:
![kk1qdkm](https://cloud.githubusercontent.com/assets/11185738/23102017/713c2fca-f6a1-11e6-9bcb-857e210291cd.png)
![rabinxs](https://cloud.githubusercontent.com/assets/11185738/23102018/82d1b1ec-f6a1-11e6-88b1-4a9b5ec65f8e.png)
![vax7zw9](https://cloud.githubusercontent.com/assets/11185738/23102019/8ff9a744-f6a1-11e6-9704-96912b255e1b.png)
![tx6csls](https://cloud.githubusercontent.com/assets/11185738/23102021/97946156-f6a1-11e6-889c-1bf10663cdf7.png)

I'm also linking to a zipped thread dump if this is of any help:
https://mega.nz/#!MARigQzB!SxKPXkTtR2uqhZla0vLolKu4Txat7HiE1fx-qVd-eHM
BTW I had to restart MEGAsync as it wouldn't recognize the new file was added to a folder.

My system info:
[systeminfo.txt](https://github.com/meganz/MEGAsync/files/785808/systeminfo.txt)

**Comment by anacondaqq** [comment]
I think this happened because of large storage and many files in storage... I can reproduce the same CPU usage on account with many files (more than 30GB of files with average size 30-50MB)

**Comment by mateusz-bajorek** [comment]
Yes. Some context: I want to keep backup of all of my photos and also be able to push them to storage once I take a photo with my phone. I used OneDrive so far but my 100GB Bing reward has ended few days ago and I was left with your storage is full message. I considered all options and decided that MEGA looks like the best solution. I uploaded all the files using MEGAsync (let's say 30GB of photos, 3-4 MB each, and 5 GB of videos). 
I am away for a week unfortunately, when I get back I can monitor whether the CPU usage jumps so high by itself after some time or if there needs to be a new file to be uploaded/downloaded and that's when MEGAsync starts acting up.
